components/slider/Slider.razor.cs

```diff
@@ -527,23 +514,15 @@ private void ValidateParameter()
                 throw new ArgumentOutOfRangeException(nameof(Step), "Must greater than 0.");
             }
 
-            if (Step != null && (Max - Min) / Step % 1 != 0)
+            var minMaxStepComparison = (Max - Min) / Step % 1;
```

Comment:
Is this logic necessary? I don't see it in the react library and the slider library that Ant react uses doesn't bother checking for this either. Did not doing it cause a bug before?

Reply:
Maybe not.